browser/farbling/brave_screen_farbling_browsertest.cc

```diff
@@ -305,6 +305,8 @@ class BraveScreenFarblingBrowserTest : public InProcessBrowserTest {
           EXPECT_LE(parent_bounds.width(), child_bounds.width());
           EXPECT_LE(parent_bounds.height(), child_bounds.height());
         }
+        /*
+        TODO(aedelstein): Fix intermittent failures here.
```

Comment:
please create an issue and add the issue link here

Reply:
Here is the issue: https://github.com/brave/brave-browser/issues/28155

I will be working on it shortly.